check-ntpoffset/lib/check-ntpoffset.go

```diff
@@ -206,3 +211,65 @@ func parseNTPOffsetFromChrony(out io.Reader) (offset float64, err error) {
 	}
 	return 0.0, fmt.Errorf("failed to get ntp offset")
 }
+
+func getNTPStratum() (string, error) {
```

Comment:
What should this function do when `ntpServers` are specified?
Is it OK to get stratum from ntpd in such case?

Reply:
I suppose nothing to do for that case, because `ntpServers` are equal to `stratum`.